app/uk/gov/hmrc/tai/model/domain/AnnualAccount.scala

```diff
@@ -20,19 +20,14 @@ import play.api.libs.json.{Format, Json}
 import uk.gov.hmrc.tai.model.TaxYear
 
 case class AnnualAccount(
-  key: String,
+  sequenceNumber: Int,
```

Comment:
Just wondering, do we even need this in the frontend?

Reply:
I think we need it to match the Json coming from Tai backend

Reply:
Lets try removing it and see what happens

Reply:
Made this change. Its working as expected after removing. Done.